packages/api-explorer/src/components/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import React, { FC } from 'react'
+import { Space } from '@looker/components'
+import { ApixHeading } from '../common'
+
+interface NotFoundProps {
+  message?: string
+}
+
+export const NotFound: FC<NotFoundProps> = ({ message }) => (
+  <Space p="xxxlarge" width="100%" height="50vh" around>
+    <ApixHeading as="h1">
+      {message || 'The page you requested could not be found.'}
```

Comment:
Does this end up with the "not found" url being the URL that wasn't found? If not, we should show what the requested URL was, perhaps. It would also be cool to make recommendations for the correct URL, if feasible.

Reply:
It does end up with the not found url as the url. I didn't register a route for not found, but we can do that too.